lib/ansible/modules/network/panos/panos_commit.py

```diff
@@ -78,25 +140,35 @@
 
 def main():
     argument_spec = dict(
-        ip_address=dict(),
-        password=dict(no_log=True),
-        username=dict(default='admin'),
+        ip_address=dict(required=True, type='str'),
+        password=dict(fallback=(env_fallback, ['ANSIBLE_NET_PASSWORD']), no_log=True),
+        username=dict(fallback=(env_fallback, ['ANSIBLE_NET_USERNAME'])),
```

Comment:
This change will break existing playbooks if they rely on the default username being `'admin'`.

Reply:
I will add "admin" as last fallback option for backward compatibilty.